### ISSUE TITLE
webrtc-win.zip filename fail

### ISSUE THREAD

**Issue by gtk2k** [Unity-Technologies/com.unity.webrtc#89]
The file name in webrtc-win.zip is missing "_" (underscore).
Example:
media_stream_interface.h
webrtc-win.zip
   > mediastreaminterface.h

(win10 2004)

**Comment by karasusan** [comment]
Could you tell me the version of `libwebrtc` you are using?
It sounds like the version is [M79](https://github.com/Unity-Technologies/com.unity.webrtc/releases/tag/M79) not [M72](https://github.com/Unity-Technologies/com.unity.webrtc/releases/tag/M72).

`develop` branch is depending on M79.

**Comment by gtk2k** [comment]
Ah,
I used webrtc-win.zip of M72 in the develop branch.
The error disappeared when I made webrtc-win.zip of M79,
I got an error that jsoncpp.lib is missing.

**Comment by karasusan** [comment]
Yes, `jsoncpp.lib` was merged into `webrrc.lib`

**Comment by gtk2k** [comment]
thanx
